src/DurableTask.ServiceBusAMS/DurableTask.ServiceBusAMS.csproj

```diff
@@ -38,24 +38,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="ImpromptuInterface" version="6.2.2" />
-    <PackageReference Include="Microsoft.Azure.KeyVault.Core" version="1.0.0" />
-    <PackageReference Include="Microsoft.Data.Edm" version="5.8.4" />
-    <PackageReference Include="Microsoft.Data.OData" version="5.8.4" />
-    <PackageReference Include="Microsoft.Data.Services.Client" version="5.8.4" />
-    <PackageReference Include="Microsoft.WindowsAzure.ConfigurationManager" version="3.2.1" />
-    <PackageReference Include="System.Spatial" version="5.8.4" />
     <PackageReference Include="Azure.Messaging.ServiceBus" version="7.11.1" />
     <PackageReference Include="Azure.Data.Tables" Version="12.8.0" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.12.0" />
-    <PackageReference Include="Azure.Storage.Queues" Version="12.12.0" />
-    <PackageReference Include="Azure.Storage.Common" Version="12.13.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.16.0" />
```

Comment:
any specific reason behind these storage packages upgrade?

Reply:
They are a year out of date and Azure.Storage.Blobs 12.12.0  is deprecated for vulnerabilities as reported on nuget.org